src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx

```diff
@@ -1,20 +1,16 @@
 ---
 title: AWS System Manager Distributor
-tags:
-  - Integrations
-  - Amazon integrations
-  - AWS integrations list
 metaDescription: "New Relic's AWS System Manager Distributor monitoring integration: what data it reports, and how to enable it."
 ---
 
-[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for reporting your application and AWS System Manager Distributor data to New Relic observability platform. This document explains the integration's features, how to activate it, and what data can be reported.
+Quickly integrate your AWS instances with New Relic's infrastructure agent through our integration for AWS System Manager Distributor. This document explains the integrations features, how to activate it, and what data can be reported.
```

Comment:
I'd say:
"Quickly install  New Relic's infrastructure agent on your AWS instances through our integration for AWS System Manager Distributor."